tendermint/src/rpc/endpoint/abci_query.rs

```diff
@@ -111,17 +111,3 @@ pub struct AbciQuery {
     #[serde(default = "String::new")]
     pub codespace: String,
 }
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-    use serde_json;
-
-    #[test]
-    fn parse_query_responses() {
-        let error_rsp = r#"{"response": {"code": 1, "log": "account lookup failed: account not found", "info": "", "index": "0", "key": null, "value": null, "proof": null, "height": "0", "codespace": ""}}"#;
-        let success_rsp = r#"{"response": {"value": "some value"}}"#;
-        serde_json::from_str::<Response>(error_rsp).expect("parse error response");
-        serde_json::from_str::<Response>(success_rsp).expect("parse success response");
-    }
```

Comment:
why is this test removed?

Reply:
it was removed in the latest official v0.33 branch